taxi/views.py

```diff
@@ -1,3 +1,16 @@
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
+from taxi.models import Car, Driver, Manufacturer
```

Comment:
fix the imports. python modules first, then installed modules, then your own and each group should be separated by blank line